Tests/AbstractTests/BoundedSemilatticeTests.generated.swift

```diff
@@ -0,0 +1,48 @@
+// Generated using Sourcery 0.7.2 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
I think Sourcery's lack of existence in Linux means that we should check these in. You're right.